Data ingestion using SINGER 2.py

```diff
@@ -0,0 +1,32 @@
+import singer as sg
+
+candidate ={
```

Comment:
Good JSON to begin practice with. Next try working with a few JSON package methods